components/com_content/src/Helper/RouteHelper.php

```diff
@@ -105,4 +105,18 @@ public static function getFormRoute($id)
     {
         return 'index.php?option=com_content&task=article.edit&a_id=' . (int) $id;
     }
+
+    /**
+     * Get the category form route.
+     *
+     * @param   integer  $id  The category ID.
+     *
+     * @return  string  The category form route.
+     *
+     * @since   6.1.0
```

Comment:
```suggestion
     * @since   __DEPLOY_VERSION__
```

doing this means our build scripts will correctly set the version - it might not make it to 6.1 ;)